include/boost/geometry/algorithms/detail/disjoint/point_geometry.hpp

```diff
@@ -39,20 +39,25 @@ namespace detail { namespace disjoint
 {
 
 
-template<typename Point, typename Geometry>
-struct disjoint_point_linear
+struct disjoint_point_segment
 {
-    static inline
-    bool apply(Point const& pt, Geometry const& g)
+    template <typename Point, typename Segment>
+    static inline bool apply(Point const& point, Segment const& segment)
     {
-        return !geometry::covered_by(pt, g);
+        typedef geometry::model::referring_segment<Point const> other_segment;
+
+        other_segment other(point, point);
+        return disjoint_segment
+            <
+                Segment, other_segment
+            >::apply(segment, other);
```

Comment:
I think this is a new combination - the diff seems a bit off here
!geometry::covered_by is still used